README.md

```diff
@@ -2,34 +2,35 @@
 
 [![Build Status](https://travis-ci.org/dojo/widgets.svg?branch=master)](https://travis-ci.org/dojo/widgets)
 [![codecov](https://codecov.io/gh/dojo/widgets/branch/master/graph/badge.svg)](https://codecov.io/gh/dojo/widgets)
-[![npm version](https://badge.fury.io/js/@dojo/widgets.svg)](http://badge.fury.io/js/@dojo/widgets)
+[![npm version](https://badge.fury.io/js/%40dojo%2Fwidgets.svg)](https://badge.fury.io/js/%40dojo%2Fwidgets)
 
-A core widget library for Dojo.
+Provides Dojo2 core widget and mixin functionality for creating custom widgets. For Dojo2 widgets please see [@dojo/widgets](https://github.com/dojo/widgets).
```

Comment:
We should be saying Dojo 2 rather than Dojo2, to avoid weirdness when we release Dojo 3. :)